# 問題
> 質量$2.0[kg]$の球を長さ$4.9[m]$のヒモに結び，一端を天井に固定して鉛直となす角$60°$で回転させた．重力加速度の大きさを$g=9.8[m/s2]$として，回転の周期$τ[s]$を求めよ．

In [7]:
from math import sqrt, pi, cos

2*pi*sqrt((4.9*0.5)/9.8)

3.141592653589793

解き方：これは「円すい振り子（conical pendulum）」です。ヒモは鉛直から角θ＝60°傾き、一定角速度で水平円運動します。半径は (r=L\sin\theta) です。

力のつり合いと向心力より

* 鉛直方向：(T\cos\theta = mg)
* 水平方向（向心力）：(T\sin\theta = m\frac{v^2}{r})

上2式から張力 (T) を消すと
[
\frac{v^2}{r} = g\tan\theta
]
また (\frac{v^2}{r}=\omega^2 r)（(\omega) は角速度）なので
[
\omega^2 = \frac{g\tan\theta}{r}=\frac{g\tan\theta}{L\sin\theta}
= \frac{g}{L\cos\theta}.
]
したがって周期 (\tau) は
[
\tau=\frac{2\pi}{\omega}=2\pi\sqrt{\frac{L\cos\theta}{g}}.
]

数値代入（(L=4.9\ \mathrm{m},\ \theta=60^\circ,\ g=9.8\ \mathrm{m/s^2})）：
[
\tau = 2\pi \sqrt{\frac{4.9\cdot \cos 60^\circ}{9.8}}
= 2\pi \sqrt{\frac{4.9\cdot 0.5}{9.8}}
= 2\pi \sqrt{\frac{2.45}{9.8}}
= 2\pi \sqrt{0.25}
= 2\pi \cdot 0.5
= \pi\ \mathrm{s}.
]

答え：(\boxed{\tau = \pi\ \text{s} \ (\approx 3.14\ \text{s})})

※質量 (m) は最終的に消えるため、与えられていても周期には影響しません。
